relaymgr/relaymgr.go

```diff
@@ -0,0 +1,138 @@
+package relaymgr
```

Comment:
Here's a component, separate from all the deal signing protocol, that just is run in the background while `auc wallet daemon` to connect and keep a healthy connection with the relay.

Reply:
Slick!